l3kernel/l3tl.dtx

```diff
@@ -2427,13 +2427,14 @@
 % \end{macro}
 % \end{macro}
 %
-% \begin{macro}[EXP,pTF]{\tl_if_single:N}
+% \begin{macro}[EXP,pTF]{\tl_if_single:N, \tl_if_single:c}
```

Comment:
Do we need `updated = 2023-10-04` here (I just pushed https://github.com/muzimuzhi/latex3/commit/f11093c37f27c552f12eff98bca87ad2676f7d57 but this PR was already merged...) and even in the documentation part?

Reply:
No, we only do 'updated' for changes in behaviour of the base function